### PR TITLE
BUG: Allow workflows without labels to be executed with run

### PR DESCRIPTION
**Pull request by ambrosejcarr**
- Only validate labels when a label is passed, even if `validate_labels` is `True`

### CODE REVIEW

File: cromwell_tools/cromwell_tools.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ def start_workflow(
 
     :return requests.Response response: HTTP response from cromwell.
     """
-    if validate_labels:
+    if validate_labels and label is not None:
```
Comment (rexwangcc) [inline cromwell_tools/cromwell_tools.py]: Why is this necessary? I would imagine if you set `validate_labels` to True, it indicates that you will pass in a label to the workflow right?

Comment (ambrosejcarr) [reply cromwell_tools/cromwell_tools.py]: validate_labels is `True` by default, and not set by the `cromwell-tools run` command line, therefore causes the cli to fail under the default specification.

Comment (samanehsan) [reply cromwell_tools/cromwell_tools.py]: The defaults are `label=None` and `validate_labels=True`. Maybe the default for `validate_labels` should be `False` then, but I imagine you would more often want to validate that the labels are correct.

Comment (rexwangcc) [reply cromwell_tools/cromwell_tools.py]: OK, I see, sorry I missed the default value here with what we have in Lira. 

> validate_labels is `True` by default

It's defo a bug then. Nice catch!
